admin/src/test/java/com/maple/admin/service/DefaultBossItemAdminServiceTest.java

```diff
@@ -71,7 +71,7 @@ void setUp() {
     }
 
     @Test
-    void 랜덤_보스_아이템_dto가_null_이면_생성_실패() {
+    void 랜덤_보스_아이템_생성_실패__랜덤_보스_아이템_dto가_null() {
```

Comment:
```suggestion
    void 랜덤_보스_아이템_생성_실패__dto가_null() {
```

마찬가지로 `랜덤_보스_아이템` 이게 중복되는거 같습니다.